# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/transforms/rotated-clip-large.png
# Commit: 2dc0263436f7
# Full Hash: 2dc0263436f78b1689d5e23a2b24720181e91ea4
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2019-11-01 09:36:21
# Regressor Bug: 1592891
# File Overlap Count: 4
# Description:
#   Bug 1592891 - Disable picture caching when picture caches have complex transforms. r=kvark,nical
#   
#   With the recent changes to compositing in WR, the scene must either
#   only produce produce cache tiles, or disable picture caching and
#   rasterize directly.
# ==============================================================================

diff -r b96cb18aa920 -r 2dc0263436f7 gfx/wr/wrench/reftests/transforms/rotated-clip-large.png
Binary file gfx/wr/wrench/reftests/transforms/rotated-clip-large.png has changed
